htdocs/comm/propal/class/api_proposals.class.php

```diff
@@ -77,6 +77,9 @@ function get($id)
 			throw new RestException(401, 'Access not allowed for login '.DolibarrApiAccess::$user->login);
 		}
 
+		// Add billing contacts ids
+                $this->propal->billing_contacts_ids = $this->propal->liste_contact(-1,'external',1,'BILLING');
```

Comment:
Why limiting array of contacts to 'BILLING' contact. If we include contacts linked to object, why not just returning all type (it does not take more effort/time) and let the api caller keep what it needs.

Reply:
You're right. I modify to get "external" contacts instead. Ithink it's too messy with every contacts in the same list.
Let me know.